event-schema/test_input.json

```diff
@@ -0,0 +1 @@
2,"813":813,"814":814,"815":815,"816":816,"817":817,"818":818,"819":819,"820":820,"821":821,"822":822,"823":823,"824":824,"825":825,"826":826,"827":827,"828":828,"829":829,"830":830,"831":831,"832":832,"833":833,"834":834,"835":835,"836":836,"837":837,"838":838,"839":839,"840":840,"841":841,"842":842,"843":843,"844":844,"845":845,"846":846,"847":847,"848":848,"849":849,"850":850,"851":851,"852":852,"853":853,"854":854,"855":855,"856":856,"857":857,"858":858,"859":859,"860":860,"861":861,"862":862,"863":863,"864":864,"865":865,"866":866,"867":867,"868":868,"869":869,"870":870,"871":871,"872":872,"873":873,"874":874,"875":875,"876":876,"877":877,"878":878,"879":879,"880":880,"881":881,"882":882,"883":883,"884":884,"885":885,"886":886,"887":887,"888":888,"889":889,"890":890,"891":891,"892":892,"893":893,"894":894,"895":895,"896":896,"897":897,"898":898,"899":899,"900":900,"901":901,"902":902,"903":903,"904":904,"905":905,"906":906,"907":907,"908":908,"909":909,"910":910,"911":911,"912":912,"913":913,"914":914,"915":915,"916":916,"917":917,"918":918,"919":919,"920":920,"921":921,"922":922,"923":923,"924":924,"925":925,"926":926,"927":927,"928":928,"929":929,"930":930,"931":931,"932":932,"933":933,"934":934,"935":935,"936":936,"937":937,"938":938,"939":939,"940":940,"941":941,"942":942,"943":943,"944":944,"945":945,"946":946,"947":947,"948":948,"949":949,"950":950,"951":951,"952":952,"953":953,"954":954,"955":955,"956":956,"957":957,"958":958,"959":959,"960":960,"961":961,"962":962,"963":963,"964":964,"965":965,"966":966,"967":967,"968":968,"969":969,"970":970,"971":971,"972":972,"973":973,"974":974,"975":975,"976":976,"977":977,"978":978,"979":979,"980":980,"981":981,"982":982,"983":983,"984":984,"985":985,"986":986,"987":987,"988":988,"989":989,"990":990,"991":991,"992":992,"993":993,"994":994,"995":995,"996":996,"997":997,"998":998,"999":999,"type":"track","event":"Demo Track","sentAt":"2019-08-12T05:08:30.909Z","channel":"android-sdk","context":{"app":{"name":"RudderAndroidClient","build":"1","version":"1.0","namespace":"com.rudderlabs.android.sdk"},"device":{"id":"49e4bdd1c280bc00","name":"generic_x86","model":"Android SDK built for x86","manufacturer":"Google"},"locale":"en-US","screen":{"width":1080,"height":1794,"density":420},"traits":{"anonymousId":"49e4bdd1c280bc00"},"library":{"name":"com.rudderstack.android.sdk.core"},"network":{"carrier":"Android"},"user_agent":"Dalvik/2.1.0 (Linux; U; Android 9; Android SDK built for x86 Build/PSR1.180720.075)"},"rudderId":"90ca6da0-292e-4e79-9880-f8009e0ae4a3","messageId":"a82717d0-b939-47bd-9592-59bbbea66c1a","properties":{"label":"Demo Label","value":5,"testMap":{"t1":"a","t2":4},"category":"Demo Category","floatVal":4.501,"testArray":[{"id":"elem1","value":"e1"},{"id":"elem2","value":"e2"}]},"anonymousId":"anon_id","integrations":{"All":true},"originalTimestamp":"2019-08-12T05:08:30.909Z"}],"writeKey":"29grCPTQ6XrVOSK76M7vfE5BWjb","requestIP":"127.0.0.1","receivedAt":"2022-06-15T13:51:08.754+05:30"}
```

Comment:
`[minor]` It is convectional in Go to put such test payload instead a `testdata` folder

https://dave.cheney.net/2016/05/10/test-fixtures-in-go

Reply:
Good catch. 🙌🏼